src/styles/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+  border: $border-size dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  @keyframes move {
+    from {
+      transform: rotate(0);
+      transform-origin: $default-transform-origin;
+    }
+
+    to {
+      transform: rotate(360deg);
+      transform-origin: $default-transform-origin;
+    }
+  }
+
+  &__minutes-hand {
+    height: $minutes-hand-height;
+    width: $minutes-hand-width;
+    background-color: #0700ff;
+    position: absolute;
+    left: center-position(($minutes-hand-width / 2));
+    top: center-position($minutes-hand-height);
```

Comment:
The `center-position` function is used here, but ensure that the `$stopwatch-size` variable is defined, as it is used within the function to calculate positions.